region/crateria/east/West Ocean.json

```diff
@@ -592,6 +663,75 @@
         }
       }
     },
+    {
+      "link": [3, 3],
+      "name": "Leave Spinning (Space Jump)",
+      "requires": [
+        "SpaceJump"
+      ],
+      "exitCondition": {
+        "leaveSpinning": {
+          "remoteRunway": {
+            "length": 45,
+            "openEnd": 1
+          }
+        }
+      }
+    },
+    {
+      "link": [3, 3],
+      "name": "Leave With Mockball",
+      "requires": [],
+      "exitCondition": {
+        "leaveWithMockball": {
+          "remoteRunway": {
+            "length": 45,
+            "openEnd": 1
+          },
+          "landingRunway": {
+            "length": 7,
+            "openEnd": 1,
+            "steepUpTiles": 1
+          }
+        }
+      }
+    },
+    {
+      "link": [3, 3],
+      "name": "Leave With Spring Ball Bounce (Space Jump)",
+      "requires": [
+        "SpaceJump"
+      ],
+      "exitCondition": {
+        "leaveWithSpringBallBounce": {
+          "remoteRunway": {
+            "length": 45,
+            "openEnd": 1
+          },
+          "landingRunway": {
+            "length": 7,
+            "openEnd": 1,
+            "steepUpTiles": 1
+          },
+          "movementType": "uncontrolled"
```

Comment:
I missed that spacejump was here but did see that you can neutral bounce into an uncontrolled springball bounce.  Hard to control, but was fun.

Reply:
Yeah, that works. I went ahead now and added a "canInsaneJump" option for that version, with a note. I'm not sure yet if these high-speed uncontrolled bounces have any applications that wouldn't already be covered by controlled bounces, but I guess we'll find out.